appveyor.yml

```diff
@@ -0,0 +1,25 @@
+# Should we cache local modules ?
+# cache:
+#  - node_modules
+
+environment:
+  matrix:
+    - nodejs_version: "6"
+    - nodejs_version: "7"
+    - nodejs_version: "8"
+
+platform:
+  - x86
+  - x64
+
+install:
+  - ps: Install-Product node $env:nodejs_version $env:platform
+  - npm install
+
+test_script:
+  - node --version
+  - npm --version
+  - IF %nodejs_version% EQU 7 npm run travis:lint 
```

Comment:
@stof I just followed the configuration that was given in travis config file.
If we decide on not including this I can remove it